vendor/bat-native-ads/src/bat/ads/history_info.cc

```diff
@@ -49,21 +50,22 @@ bool HistoryInfo::FromValue(const base::Value::Dict& root) {
 
 std::string HistoryInfo::ToJson() const {
   std::string json;
-  base::JSONWriter::Write(ToValue(), &json);
+  bool result = base::JSONWriter::Write(ToValue(), &json);
```

Comment:
`const`

Reply:
nit: for consistency with the result of ads can we use `bool success`

Reply:
Fixed in 205d01f9223f4d8c6501fd0a63b2c24559b5b1c3.